keel-core/src/test/kotlin/com/netflix/spinnaker/keel/constraints/ManualJudgementConstraintEvaluatorTests.kt

```diff
@@ -65,6 +67,8 @@ internal class ManualJudgementConstraintEvaluatorTests : JUnit5Minutests {
   fun tests() = rootContext<Fixture> {
     fixture { Fixture }
```

Comment:
```suggestion
    fixture { Fixture() }
```

Reply:
and change `object Fixture` above to `class Fixture` (can't figure out how to make suggestions on lines you didn't change in the PR)

Reply:
oooo but that fixes my mocks problem! interesting